### PR TITLE
FSA22V2-273: Accordion code example

### PR DESCRIPTION
**Pull request by jkohrt7**
### Description:

- Add HTML, CSS and JS for the WCAG-compliant accordion component example

The accordion being added in this PR is meant to serve as a bare-bones template for developers. It should meet the WCAG guidelines described for the accordion design pattern and have simple, understandable javascript controlling it. The styling is intentionally minimal so that it can be easily expanded upon and customized for any project.

### Spec:

CodePen: [Accordion](https://codepen.io/jared-kohrt/pen/bGKRxQV)

See Story: [FSA22V2-273](https://sparkbox.atlassian.net/browse/FSA22V2-273)

### Validation:

- [ ] This PR has code changes, and our linters still pass.
- [ ] HTML, CSS, and JS matching the codepen is displayed on the /accordion route in the correct areas.

#### To Validate:

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
2. Pull down `FSA22V2-273-integration-accordion-code-example`.
3. Run `npm install` to make sure you have all proper dependencies. (this project requires Node 16+)
4. Copy the `.env.example` file, making sure it's in the root of your project, and rename it `.env.local`. Search for "Accessible Components" in 1Password to find the API Key and Base ID. Add both of those secrets to your newly created `.env.local` file. That should establish your connection to our Airtable database.
5. Run `npm run lint` to confirm no errors.
6. Run `npm run test` to confirm all tests pass.
7. In terminal: `npm run dev`.
8. Check [localhost:3000/accordion](http://localhost:3000/accordion) to see changes.

### CODE REVIEW

File: src/styles/components/_details-page.scss [hunk 1]
```diff
@@ -15,6 +15,12 @@
     width: 100%;
     margin: auto;
     background-color: map.get($colors, "accent-teal");
+
+    &:last-child {
+      @media (prefers-color-scheme: dark) {
+        background-color: var(--background-color);
+      }
+    }
```
Comment (dustin-jw) [inline src/styles/components/_details-page.scss:23]: Is this doing anything? I think this is already handled by `.details-page__secondary--related-components`.

**Review by dustin-jw — CHANGES_REQUESTED** [review]
I think we can omit the styling change you added, but otherwise the code sample looks good and worked when I copied it to CodePen!

File: src/styles/components/_details-page.scss (hunk 1 above)
Comment (jkohrt7) [reply src/styles/components/_details-page.scss:23]: Yes, it fixes a problem with the green bg appearing to the sides of the component inventory in dark mode.

Without that line of code:
Screen Shot 2022-11-18 at 4 32 45 PM
vs with:
image

Comment (dustin-jw) [reply src/styles/components/_details-page.scss:23]: Oh I see, gotta get to those really wide screen sizes! Okay, all good then!

**Review by dustin-jw — DISMISSED** [review]

**Review by dustin-jw — DISMISSED** [review]

**Comment by jkohrt7** [comment]
Had to rebase on main again for the dependencies fix 😅

**Review by dustin-jw — APPROVED** [review]
Okay for real this time 😂

**Pull request merged**